bootloader/Vagrantfile

```diff
@@ -144,23 +221,83 @@ Vagrant.configure(2) do |config|
+        :inline => "chown -R vagrant: /vagrant || true"
     b.vm.provision "packages debianoid", :type => :shell, :inline => packages_debianoid
-    b.vm.provision "build bootloader", :type => :shell, :privileged => false, :inline => build_bootloader("linux64")
+    b.vm.provision "build bootloader", :type => :shell, :privileged => false, :inline => build_bootloader("linux64"),
+        :keep_color => true
+    b.vm.provision "build bootloader for windows", :type => :shell,
+        :privileged => false,
+        :inline => build_bootloader_target_win32("linux64"),
+        :keep_color => true
+    if File.exist?("_macOSX/osxcross.tar.bz2")
+      # Do not rsync this to avoid copying 4GB into the machine
+      config.vm.synced_folder "_macOSX", "/vagrant/macOSX",
+          :mount_options => ["ro"]
+      b.vm.provision "build bootloader for osx", :type => :shell,
+          :privileged => false,
+          :inline => build_bootloader_target_osx("linux64"),
+          :keep_color => true
+    end
+  end
+
+  #--- Build the Darwin SDK and tools ---
+  config.vm.define "build-osxcross" do |b| # FIXME: Rename
+    b.vm.box = "boxcutter/debian82"
+    b.vm.provider :virtualbox do |v|
+      v.memory = 768
+    end
+
+    if File.exist?("_macOSX/Xcode_7.3.1.dmg")
+      # Do not rsync this to avoid copying 4GB into the machine
+      config.vm.synced_folder "_macOSX", "/vagrant/macOSX",
+          :mount_options => ["ro"]
+    end
+
+    b.vm.provision "packages osxcross debianoid", :type => :shell,
+        :inline => packages_osxcross_debianoid,
+        :keep_color => true
+    b.vm.provision "prepare osxcross debianoid", :type => :shell,
+        :inline => prepare_osxcross_debianiod,
+        :keep_color => true
+    # Reload for the case the kernel changed, thus fuse will be loaded
+    b.vm.provision :reload
+    b.vm.provision "build osxcross", :type => :shell,
+        :inline => build_osxcross,
+        :keep_color => true
   end
 
   #--- OS X --- not yet implemented
   config.vm.define "darwin64" do |b|
     b.vm.box = "jhcook/yosemite-clitools"
     b.vm.provider :virtualbox do |v|
       v.gui = true
-      # Enable the VM's virtual USB controller & enable the virtual
-      # USB 2.0 controller
-      v.customize ["modifyvm", :id, "--usb", "on", "--usbehci", "off"]
+      v.customize ['modifyvm', :id, '--ostype', 'MacOS1010_64']
+      v.customize ['modifyvm', :id, '--paravirtprovider', 'default']
+      # Adjust CPU settings according to
+      # https://github.com/geerlingguy/macos-virtualbox-vm
+      v.customize ['modifyvm', :id, '--cpuidset',
+                   '00000001', '000306a9', '00020800', '80000201', '178bfbff']
+      # This should make make the box less choppy (source: geerlingguy)
+      #v.customize ["modifyvm", :id, '--audio', 'off']
+      # Disable USB variant requiring Virtualbox proprietary extension packs
+      v.customize ["modifyvm", :id, '--usbehci', 'off', '--usbxhci', 'off']
```

Comment:
As far as I can toll, yes. On one of my systems there is only a "DTrace" extension pack, and the machine starts.